lib/matplotlib/axes/_axes.py

```diff
@@ -6553,10 +6553,10 @@ def hist2d(self, x, y, bins=10, range=None, normed=False, weights=None,
 
         Parameters
         ----------
-        x, y: array_like, shape (n, )
+        x, y : array_like, shape (n, )
             Input values
 
-        bins: [None | int | [int, int] | array_like | [array, array]]
+        bins : None or int or [int, int] or array_like or [array, array]
```

Comment:
(int, int) or (array, array)?  (feels like tuples, nor arrays)

Reply:
This description came from 
[numpy.histogram2d](https://docs.scipy.org/doc/numpy-1.14.0/reference/generated/numpy.histogram2d.html#numpy.histogram2d). I would leave it like this.